assets/js/components/OutfrontTakeoverTool/AlertDashboard/AlertDetails.tsx

```diff
@@ -65,7 +65,7 @@ const AlertDetails = (props: AlertDetailsProps): JSX.Element => {
   );
 
   return (
-    <div className="alert-card">
+    <div className="oft-alert-card">
```

Comment:
You might need to merge in changes from the feature branch one more time. This was a change from one of my commits that I ended up changing before merging.

Reply:
I think I merged poorly. Hope nothing else got through 😓